lib/zhr_devs/identity_management/event_handler.ex

```diff
@@ -0,0 +1,39 @@
+defmodule ZhrDevs.IdentityManagement.EventHandler do
+  @moduledoc """
+  This module is responsible for handling events.
+  Usually used to perform a side effect, such as sending an email or updating a read model.
+  In our case it will be responsible to maintain our in-memory projection of an identity.
```

Comment:
I understand that projection is a fold of events? But again, copy pasta / links would be cute. (Sorry for making you do that, but it will greatly improve onboarding, because reading huge amounts of docs can be overwhelming for many people).